grails-app/services/au/org/ala/bie/ImportService.groovy

```diff
@@ -88,6 +88,11 @@ class ImportService implements GrailsConfigurationAware {
             DwcTerm.vernacularName,
             ALATerm.nameID,
             DwcTerm.kingdom,
+            DwcTerm.phylum,
+            DwcTerm.class_,
```

Comment:
Just checking `class_` is not a typo as the other values do not have `_` suffix

Reply:
Yes. DwC.class_ corresponds to http://rs.tdwg.org/dwc/terms/class but, because class is already taken by Java, a _ is added to the end by GBIF. Not a bad convention, easier to navigate than clazz.

Reply:
Oh yeah - java reserved word - forgot about that.